app/models/collected-metadatum.ts

```diff
@@ -53,10 +53,13 @@ export default class CollectedMetadatumModel extends OsfModel.extend(Validations
 
     @computed('collection.displayChoicesFields.[]')
     get displayChoiceFields() {
-        return choiceFields
-            .filter(field => this.collection
-                .get('displayChoicesFields')
-                .includes(`${field}Choices` as ChoicesFields));
+        if (this.collection.get('displayChoicesFields')) {
```

Comment:
can we do without the `.get` here?

Reply:
In this case, `collection` is still a proxy, so we'll have to use the `.get` here.

Reply:
ah proxies 😩 Depending on a async relationship is best practices but that can be addressed later.